distribution/packages/src/common/scripts/postinst

```diff
@@ -49,20 +49,68 @@ case "$1" in
         exit 1
     ;;
 esac
-
 # to pick up /usr/lib/sysctl.d/elasticsearch.conf
 if command -v systemctl > /dev/null; then
     systemctl restart systemd-sysctl.service || true
 fi
-
 if [ "x$IS_UPGRADE" != "xtrue" ]; then
+    # Don't exit immediately on error, we want to hopefully print some helpful banners
+    set +e
+    # Attempt to auto-configure security, this seems to be an installation
+    if ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.ConfigInitialNode \
+    ES_ADDITIONAL_SOURCES="x-pack-env;x-pack-security-env" \
+    ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli \
+    /usr/share/elasticsearch/bin/elasticsearch-cli -strict <<< ""; then
```

Comment:
TIL that you can specify long options with a single dash.

Reply:
OptionsParser works like that. The fact that we add a short and a long option and we document the short with a single dash, and the long with double dash is simply convention. Anything you specify with `OptionParser#accepts` and `OptionParser#acceptsAll` can be used with both single and double dash